### PR TITLE
fix: [RGOeX-1240] save scroll position on exit from fullscreen

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** save scroll position on exit from video xblock fullscreen mode

**Youtrack:** https://youtrack.raccoongang.com/issue/RGOeX-1240

**Note:**
This PR should be merged after merging the corresponding PR in upstream MEF repos
[maple MFE Learning] https://github.com/openedx/frontend-app-learning/pull/982
[nutmeg MFE Learning] https://github.com/openedx/frontend-app-learning/pull/981
[master MFE Learning] https://github.com/openedx/frontend-app-learning/pull/983

### CODE REVIEW

File: video_xblock/backends/base.py [hunk 1]
```diff
@@ -224,7 +224,8 @@ def get_frag(self, **context):
         js_files += [
             'static/js/videojs/videojs-tabindex.js',
             'static/js/videojs/toggle-button.js',
-            'static/js/videojs/videojs-event-plugin.js'
+            'static/js/videojs/videojs-event-plugin.js',
+            'static/js/videojs/fullscreen-extends.js'
```
Comment (idegtiarov) [inline video_xblock/backends/base.py:228]: Could you please add a trailing comma to the last list element?

File: video_xblock/backends/brightcove.py [hunk 2]
```diff
@@ -433,6 +433,7 @@ def get_frag(self, **context):
             'static/js/videojs/toggle-button.js',
             'static/js/videojs/videojs-event-plugin.js',
             'static/js/videojs/brightcove-videojs-init.js',
+            'static/js/videojs/fullscreen-extends.js'
```
Comment (idegtiarov) [inline video_xblock/backends/brightcove.py:436]: ditto

**Review by idegtiarov — CHANGES_REQUESTED** [review]
LGTM, but two minor nits are mentioned in the comments inline. Please take a look.

File: video_xblock/backends/brightcove.py (hunk 2 above)
Comment (ihor-romaniuk) [reply video_xblock/backends/brightcove.py]: also

File: video_xblock/backends/base.py (hunk 1 above)
Comment (ihor-romaniuk) [reply video_xblock/backends/base.py]: have been added

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**